doc/OnlineDocs/contributed_packages/sensitivity_toolbox.rst

```diff
@@ -21,73 +24,65 @@ We will start with a motivating example:
 
 Here :math:`x_1`, :math:`x_2`, and :math:`x_3` are the decision variables while :math:`p_1` and :math:`p_2` are parameters. At first, let's consider :math:`p_1 = 4.5` and :math:`p_2 = 1.0`. Below is the model implemented in Pyomo.
 
-.. doctest::
-
-    Import Pyomo and sipopt from the sensitivity toolbox
+.. doctest:: python
 
+    # Import Pyomo and sipopt from the sensitivity toolbox
     >>> from pyomo.environ import *
-    >>> from pyomo.contrib.sensitivity_toolbox.sens import sipopt
+    >>> from pyomo.contrib.sensitivity_toolbox.sens import sensitivity_calculation
 
-    Create a concrete model
+    >>> sipopt_available = SolverFactory("ipopt_sens", validate=False).available()
+    >>> k_aug_available = SolverFactory("k_aug", validate=False).available()
+    >>> dot_sens_available = SolverFactory("dot_sens", validate=False).available()
 
```

Comment:
Added variables for executable availability and am trying to use them in a `:skipif:` block, but am getting an odd error.

Reply:
The availability flags need to be declared in the `conf.py` file to be used in the `:skipif:` block. See: https://github.com/Pyomo/pyomo/blob/main/doc/OnlineDocs/conf.py#L266